commands/app.go

```diff
@@ -141,6 +141,10 @@ func appAction(ctx *cli.Context) error {
 
 	cfg, err := config.LoadConfiguration(configFile)
 	if err != nil && token == "" {
+		fmt.Println("step-ca can't find or open the configuration file for your CA.")
+		fmt.Println("You may need to create a CA first by running `step ca init`.")
+		fmt.Println("Documentation: https://u.step.sm/docs/ca")
+		fmt.Println("")
```

Comment:
It might be better to print only this if the file does not exists or cannot be opened for any reason:
```suggestion
		var pathErr *os.PathError
		if errors.As(err, &pathErr) {
			fmt.Println("step-ca can't find or open the configuration file for your CA.")
			fmt.Println("You may need to create a CA first by running `step ca init`.")
			fmt.Println("Documentation: https://u.step.sm/docs/ca")
			os.Exit(1)
		}
```